_includes/pbc/all/install-features/202307.0/install-the-product-rating-and-reviews-feature.md

```diff
@@ -1,201 +1,1212 @@
 
 
-## Prerequisites
+This document describes how to integrate the [Product Rating and Reviews](/docs/pbc/all/ratings-reviews/{{page.version}}/ratings-and-reviews.html) feature into a Spryker project.
 
-To prepare your project to work with Product Reviews:
```

Comment:
So from now on, we don't need the prerequisites section for some reason?

Reply:
This is an old way. Now we have another Prerequisites section